File: codex_poting/java/src/main/java/com/example/demo/api/request/LoginRequest.java
package com.example.demo.api.request;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotBlank;
import lombok.Value;

@Value
public class LoginRequest {
    @NotBlank
    @Schema(description = "ユーザー名", type = "string")
    String username;

    @NotBlank
    @Schema(description = "パスワード", type = "string")
    String password;
}

File: codex_poting/kotlin/src/main/kotlin/com/example/demo/api/request/LoginRequest.kt
package com.example.demo.api.request

import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotBlank

data class LoginRequest(
    @field:NotBlank
    @field:Schema(description = "ユーザー名", type = "string")
    val username: String?,

    @field:NotBlank
    @field:Schema(description = "パスワード", type = "string")
    val password: String?
)
